constructor/header.sh

```diff
@@ -479,7 +479,7 @@ mkdir -p "$TMP"
 # but we haven't created $PREFIX/pkgs yet... give it a temp location
 # shellcheck disable=SC2050
 if [ "__VIRTUAL_SPECS__" != "" ]; then
-    echo 'Checking virtual specs compatibility: __VIRTUAL_SPECS__'
+    echo "Checking virtual specs compatibility:" __VIRTUAL_SPECS__
```

Comment:
```suggestion
    echo "Checking virtual specs compatibility: __VIRTUAL_SPECS__"
```

The new code looks like it would render as "compatibility:'__virtual'". I think just replacing the single quotes with double quotes will be fine.

Reply:
`__VIRTUAL_SPECS__` is `shlex.join`'ed so it will contain single quotes if it has `>` in it. But I'm not sure what will happen if the user types `"__osx>=10.13"`. Given that `echo` admits several parameters, I'm inclined to let it roam freely because it's supposed to work like that.